configs/francoischalifour.json

```diff
@@ -3,9 +3,6 @@
   "start_urls": [
     "https://francoischalifour.com/"
   ],
-  "sitemap_urls": [
```

Comment:
This one using a wrong RSS file — I removed it.

Reply:
Would be nice to implement one at some point. (Especially if you want to have a good SEO)
Fine for us